docs/tutorial/printing.md

```diff
@@ -140,6 +140,16 @@ In general, **Typer** tends to be the entry point to your program, taking the fi
 
 The best results for your command line application would be achieved combining both **Typer** and **Rich**.
 
+### If you don't want to use Rich formated output
+
+In case you'd rather like to use the simple text ouput, but need **Rich** to be installed in your environment for any other reason, you can disable the use of **Rich** by **Typer**.
+There are two functions in the **Typer** module to control the output format globally, affecting all `typer.Typer()` instances:
```

Comment:
```suggestion
### Configuring Rich formatted output

By default, **Typer** uses the **Rich** library to format text output.  However it's an optional dependency, and is used only if it is installed in your environment.  In case you would rather have finer-grained control over when **Typer** uses rich formatting, you can do so with the two package level functions shown below to control the output format globally.  This affects all `typer.Typer()` instances.
```
Since the new version of the global functions are setters instead of a toggle, the documentation should reflect that

Reply:
I commited your suggestions.
> I think it could be improved a bit by merging with the previous heading "Typer and Rich" and making it more coherent. As it is now, it kind of sticks out.

The previous heading speaks about the roles of `Typer` and `Rich` respectively. There might be a way to neatly integrate it there and make it more coherent but on the other hand it might be more discoverable the way it is. I think that is a decision we can leave up to the maintainer.

Reply:
okay fair point, I'll approve this